files-community/Files
# Code Quality: Fixed issues with the properties window components

## Summary

During removing SettingsViewModel, I had to alter ThemeHelper since that ViewModel contained theme change event handler. And while I changing ThemeHelper, I noticed there were several visual issues therefore I fixed in this PR.

## Task Checklist

- [x] Removed the duplicated SettingsViewModel
- [x] Fixed an issue where the action buttons of the properties window were invisible
- [x] Fixed an issuw where the window icon was not set properly

## Known Issues

Following issues would be fixed in the upcoming PR.

_N/A_

## PR Checklist

- [x] **Related**: #4180

## Steps To Validate Changes

  - Open the app
  - Open properties window
  - See x button visibility on the active window
  - And hover the Files icon on the TaskBar
  - See the icon is set properly

## Screenshots

![image](https://github.com/files-community/Files/assets/62196528/ccddb80f-50a0-43ec-bedf-d72deadf07ba)

![image](https://github.com/files-community/Files/assets/62196528/2408652b-3cad-4597-addd-8903fa1e8e96)

[^1]: If the request hasn't been agreed by the team, this work might be rejected. Make sure that you get approval from the team before opening any PR related the request.
[^2]: If you removed any en-US string resources, make sure that there are no references of those resources. When you add a new en-US string resources, do not make any changes on other languages' resources; [Crowdin](https://crowdin.com/project/files-app) will do, instead.
